services/jenkins/jenkins-coverage.tester.js

```diff
@@ -66,9 +60,17 @@ t.create('code coverage API v1: job found')
   )
   .expectBadge({ label: 'coverage', message: isIntegerPercentage })
 
-t.create('code coverage API v4+: job not found')
+t.create('code coverage API v1: job not found')
   .get(
-    '/apiv4.json?jobUrl=https://jenkins.library.illinois.edu/job/does-not-exist',
+    '/apiv1.json?jobUrl=http://loneraver.duckdns.org:8082/job/does-not-exist',
```

Comment:
I can't find any examples of a jenkins server supporting apiv1 so I've just replaced the not found test with a mock. The live test for this case is already a mock anyway.